burn-tensor/src/tensor/element.rs

```diff
@@ -18,29 +19,63 @@ pub trait Element:
 {
 }
 
+/// Element conversion trait for tensor.
 pub trait ElementConversion {
+    /// Converts an element to another element.
+    ///
+    /// # Arguments
+    ///
+    /// * `elem` - The element to convert.
+    ///
+    /// # Returns
+    ///
+    /// The converted element.
     fn from_elem<E: ToPrimitive>(elem: E) -> Self;
+
+    /// Gets the element.
```

Comment:
This is not just a get, this does the conversion as well. It can be implemented for a numeric type (`i32`) and you can call `elem::<f32>()` to convert it to a float.